content/releases.md

```diff
@@ -80,13 +80,16 @@ by `<major>.<minor>`. Releases branches will be in one of several states:
 
 - __*Next*__: The next planned release branch.
 - __*Active*__: The release is currently supported and accepting patches.
+- __*Extended*__: The release is only accepting security patches.
```

Comment:
/s/release/release branch/
and the line above.. and a few below. Alternatively we could explicitly explain the relationship between the term "release" and "release branch".

Reply:
These changes should be made on the original doc, this is only resolving a diff.